src/life.rs

```diff
@@ -891,16 +892,26 @@ impl WorldState {
 
 }
 
+fn next_color(color: Color) -> Color {
+    match color {
+        Color::Yellow => Color::Cyan,
+        Color::Cyan => Color::Yellow,
+        _ => unreachable!()
+    }
+}
```

Comment:
I couldn't figure out how to spell it... it was yelling at me about how Color doesn't have Clone defined on it. i can try again, tho.